Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderThumbnailView.cpp

```diff
@@ -521,20 +530,25 @@ namespace AzQtComponents
 
     void AssetFolderThumbnailView::setSelection(const QRect& rect, QItemSelectionModel::SelectionFlags flags)
     {
-        if (!model())
-        {
-            return;
-        }
+        // Commenting out this part for now as it doesn't work as expected and it leads to misbehaviour when accidentally
+        // triggered (i.e. when clicking on an item and slightly moving the mouse, generating an area selection rather than a click)
+        Q_UNUSED(rect);
+        Q_UNUSED(flags);
 
-        const auto translatedRect = rect.translated(horizontalOffset(), verticalOffset());
+        // if (!model())
+        // {
+        //     return;
+        // }
 
-        for (auto it = m_itemGeometry.constBegin(); it != m_itemGeometry.constEnd(); ++it)
-        {
-            if (it.value().intersects(translatedRect))
-            {
-                selectionModel()->select(it.key(), flags);
-            }
-        }
+        // const auto translatedRect = rect.translated(horizontalOffset(), verticalOffset());
+
+        // for (auto it = m_itemGeometry.constBegin(); it != m_itemGeometry.constEnd(); ++it)
+        // {
+        //     if (it.value().intersects(translatedRect))
+        //     {
+        //         selectionModel()->select(it.key(), flags);
+        //     }
+        // }
```

Comment:
Would prefer to delete this code instead of commenting it out, we can always look back to previous versions in the git history.

Reply:
Done